basic/airdrop/src/lib.rs

```diff
@@ -0,0 +1,40 @@
+use scrypto::prelude::*;
+
+blueprint! {
+    struct Airdrop {
+        recipients: Vec<Address>,
+    }
+
+    impl Airdrop {
+        pub fn new() -> Component {
+            Self {
+                recipients: Vec::new(),
+            }
+            .instantiate()
+        }
+
+        pub fn add_recipient(&mut self, recipient: Address) {
```

Comment:
Really nice example ! I think badges for authentication would be a good use case for this component. Allowing only people presenting a certain badge to call `add_recipient` and `perform_airdrop`

Reply:
Thanks for the feedback! Initially I wanted to keep the example as lean as possible. When you think about it, though, it makes sense for all examples to be properly secured. I have now added authorization to both methods as per your suggestion.